### ISSUE TITLE
Link broken

### ISSUE THREAD

**Issue by AndreaMargit** [CLARIAH/clariah-plus#139]
https://github.com/CLARIAH/clariah-plus/blob/main/technical-committee/shared-development-roadmap/epics/shared/fair-tool-discovery.md

**Comment by proycon** [comment]
Right, this one moved to https://github.com/CLARIAH/clariah-plus/blob/main/technical-committee/shared-development-roadmap/epics/fair-tool-discovery.md

Where did you find the original link? Then I might correct it.
